oclint-rules/test/naming/LongVariableNameRuleTest.cpp

```diff
@@ -46,11 +46,11 @@ TEST_F(LongVariableNameRuleTest, ThreeCharsName)
 TEST_F(LongVariableNameRuleTest, FourCharsName)
 {
     testRuleOnCode(new LongVariableNameRule(), "void aMethod(int iiii) {}",
-        0, 1, 14, 1, 18, "Variable name with 4 characters is longer than the threshold of 3");
+        0, 1, 14, 1, 18, "Variable name with 4 characters (iiii) is longer than the threshold of 3");
```

Comment:
Do you think a pair of backtick "`" might be better than parenthesis? 

And I am also thinking to make it "Length of variable name `iiii` is 4, which is longer than threshold of 3".

I am adding the comment here, but it should apply to all changes.